### DISCUSSION TITLE
Too easy to delete a task.  No confirmation, no history.

### DISCUSSION THREAD

**Discussion by CPUGPU** [ransome1/sleek#477] [General]
If a task has "focus", hitting delete on the keyboard deletes it immediately with no confirmation and no history.
If I create a new task and then immediately hit delete, the task above the new task is deleted without confirmation and with no history.  (This is V 1.2.9.  Nervous to upgrade.)

Is there a setting I'm missing?

**Comment by ransome1** [comment]
You might be right on this. There was also another bug report from a Windows user, who was using Ctrl+Alt+Delete to open his task manager and that was accidentally deleting todos, when sleek was open. I will put some effort into this and improve it. Also thinking about introducing a confirmation prompt.
